### ISSUE TITLE
Feature Request and Possible glitch

### ISSUE THREAD

**Issue by WillowMist** [pcal43/copper-hopper#20]
I'm having trouble with filtering Water buckets and milk buckets.  They only seem to sort some of the time.  Additionally, a couple of feature requests:

Possible to group items in a config file so that accepting one item would accept others in the group.  This would allow for filtering all wool, or all terracotta, or carpet, what have you.

Also, if Suspicious Stew would all filter together.  This could possibly be achieved with the above, but I'm not sure.

**Comment by WillowMist** [comment]
Oh, I see why the bucket one isn't working.  Will have to rethink a couple of my stacks.  But my feature request stands :)

**Comment by pcal43** [comment]
Hi. Thanks for the note.  I'll definitely keep it in mind.

One issue I see is that different people will probably have different ideas about how things should be grouped, and I have some desire to keep the mod as simple and vanilla-like as possible.

On a related note: you might want to look at another mod I've written:
  https://modrinth.com/mod/amazing-chest

which you can also use to do filtering but with an entire chests' worth of  slots.

**Comment by WillowMist** [comment]
Yeah, I see what you mean.  It would be fairly easy to do it at the server level, but then players may have different ideas of grouping.  I looked at the chest, and it definitely looks cool.  I'm still trying to wrap my head around how it works, at least as pictured :)

**Comment by EG0611** [comment]
Would it be possible to have NeoForge support?
